src/components/EmptySelectionListContent.tsx

```diff
@@ -39,7 +39,7 @@ function EmptySelectionListContent({contentType}: EmptySelectionListContentProps
     );
 
     return (
-        <View style={[styles.flex1, styles.overflowHidden]}>
+        <View style={[styles.flex1, styles.overflowHidden, styles.minHeight65]}>
```

Comment:
This min-height without ScrollView causes the issue https://github.com/Expensify/App/issues/53689